### PR TITLE
Bugfix/lib external helpers undefined

### LINKED ISSUE #430: with-react only avaiable as src. should be transpiled for dist
## Are you reporting a bug or a feature?

 - [ ] Bug
 - [x] Feature

## Expected Behavior

with-react only avaiable as src. should be transpiled for dist

### PR DESCRIPTION
**Pull request by AndyOGo**
fixes #430 

Fixes `babelHelpers` undefined

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "build-icons-prod": "node stack/tasks/icons.js PROD",
     "build-dev": "node stack/tasks/cleanup.js DEV && npm run build-css-dev && npm run build-html-dev && npm run build-js-dev && npm run build-icons-dev",
     "build": "node stack/tasks/cleanup.js PROD && node stack/tasks/build.html.js PROD && node stack/tasks/build-examples.js PROD && npm run build-css-prod && npm run build-js-prod && npm run build-lib && npm run build-icons-prod && node stack/tasks/copy-dist-to-docs.js",
-    "build-lib": "babel src --out-dir lib --ignore src/demos",
+    "build-lib": "babel src --out-dir lib --ignore src/demos --plugins transform-runtime",
```
Comment (AndyOGo) [inline package.json]: @LucaMele 
@pavestru 
For rollup we used the [`runtimeHelpers` option of rollup-plugin-babel](https://github.com/rollup/rollup-plugin-babel#helpers)
But we should really clean this up and have all within config files, not custom config overwrites in JS

https://github.com/axa-ch/patterns-library/search?q=runtimeHelpers&unscoped_q=runtimeHelpers

**Comment by AndyOGo** [comment]
still getting:

```sh
TypeError: WebComponent is not a constructor
```

for:

```js
var node = new WebComponent();
```

File: package.json (hunk 1 above)
Comment (LucaMele) [reply package.json]: generally, the whole build need to be cleanup up. it was a quick solution for a limited scope. :)

Comment (AndyOGo) [reply package.json]: Yeah, it really takes more time to craft a stable build and document all of it - not meant as accusation

Comment (LucaMele) [reply package.json]: totally agree :)

**Comment by AndyOGo** [comment]
@LucaMele 
@pavestru 
Since we still get this **TypeError** of `WebComponent is not a constructor` we need to find another way of getting the `tagName` used by `withReact`.
I will create another issue and PR for this.

**Comment by LucaMele** [comment]
@AndyOGo use the plugins: `"transform-custom-element-classes", "transform-class-properties",` and test again

**Comment by LucaMele** [comment]
`"build-lib": "babel src --out-dir lib --ignore src/demos --plugins transform-runtime transform-custom-element-classes transform-class-properties",`

**Comment by LucaMele** [comment]
ok yes. weird

**Comment by AndyOGo** [comment]
Yeah, I thought this at first to, but we actually use them already.

It's again this fu**** issue of asynchronous vs. synchronous registering, instantiation and rendering...

**Pull request merged**